packages/react-components/src/components/ImageGallery.tsx

```diff
@@ -23,10 +23,9 @@ import {
   titleBarContainerStyle,
   titleStyle
 } from './styles/ImageGallery.style';
-import { useTheme } from '../theming/FluentThemeProvider';
-import { isDarkThemed } from '../theming/themeUtils';
 /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
 import { useLocale } from '../localization';
+import { darkTheme } from '../theming';
```

Comment:
We probably don't want to hardcode the darktheme here.
Instead, use the `useTheme()` hook, so that if the theme is overridden then it picks up the values from the current active theme.